x-pack/plugins/alerting/common/routes/maintenance_window/apis/find/schemas/v1.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { schema } from '@kbn/config-schema';
+import { maintenanceWindowResponseSchemaV1 } from '../../../response';
+
+const MAX_DOCS_PER_PAGE = 10000;
+
+export const findMaintenanceWindowsRequestQuerySchema = schema.object(
+  {
+    page: schema.maybe(
+      schema.number({
+        defaultValue: 1,
+        min: 1,
+        max: MAX_DOCS_PER_PAGE,
+        meta: {
+          description: 'The page number to return.',
+        },
+      })
+    ),
+    per_page: schema.maybe(
+      schema.number({
+        defaultValue: 20,
```

Comment:
super nit: Could you please move all the numbers of `min, max and defaultValue` to const same as `MAX_DOCS_PER_PAGE ` for consistency?